pkgs/create-neon/README.md

```diff
@@ -6,6 +6,42 @@ The `create-neon` tool bootstraps [Neon](https://neon-bindings.com) projects, wh
 
 You can conveniently use this tool with the [`npm init`](https://docs.npmjs.com/cli/v7/commands/npm-init) syntax:
 
+### Creating a Simple Project
+
+To create a simple Neon project that consists purely of Rust code:
+
+```sh
+$ npm init neon [<opts> ...] my-project
+```
+
+#### Global Options
+
+```sh
+-y|--yes  Skip interactive `npm init` questionnaire.
+```
+
+### Creating a Portable Library
+
+To create a portable npm library with pre-built binaries:
```

Comment:
Nit, it might not be obvious to a potential user what "portable" means.

Reply:
Great point--I added some explanatory text.